pkg/kubelet/configmap/configmap_manager_test.go

```diff
@@ -18,30 +18,27 @@ package configmap
 
 import (
 	"fmt"
-	"reflect"
 	"strings"
-	"sync"
 	"testing"
 	"time"
 
 	"k8s.io/api/core/v1"
-	"k8s.io/client-go/kubernetes/fake"
 
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/clock"
-	core "k8s.io/client-go/testing"
 
-	"github.com/stretchr/testify/assert"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/kubernetes/pkg/kubelet/util/manager"
 )
 
-func checkConfigMap(t *testing.T, store *configMapStore, ns, name string, shouldExist bool) {
+func checkObject(t *testing.T, store manager.Store, ns, name string, shouldExist bool) {
```

Comment:
Nit: this could be added as a testing helper to the `manager` package to be shared by both secret and configmap managers.